apiapp.go

```diff
@@ -658,7 +658,7 @@ func checkEnv(appname string) (apppath, packpath string, err error) {
 	gopath := gps[0]
 
 	logger.Warn("You current workdir is not inside $GOPATH/src")
-	logger.Debugf("GOPATH: %s", gopath)
+	logger.Debugf("GOPATH: %s", __FILE__(), __LINE__(), gopath)
```

Comment:
No. The args for a formatted message are the last ones. This is the method's signature:
`logger.Debugf(message string, file string, line int, args ...interface{})`

`args` will passed to `fmt.Sprint()` along with the message.

Question: What does `checkEnv()` do? I'm thinking to refactor it.